### PR TITLE
fix(sql): added auto-database create

### PR DESCRIPTION
**Pull request by konijn-x**
Creates an database automatic.

### CODE REVIEW

File: [SQL]/legacy.sql [hunk 1]
```diff
@@ -1,3 +1,12 @@
+CREATE DATABASE IF NOT EXISTS `esx_legacy`;
+USE `esx-legacy`;
```
Comment (PsychoShock) [inline [SQL]/legacy.sql:2]: USE `esx_legacy`;


Fix typo :D

**Review by PsychoShock — CHANGES_REQUESTED** [review]

**Review by 0xDEMXN — APPROVED** [review]

**Comment by Mycroft-Studios** [comment]
This *may* Break the TxAdmin recipe

**Comment by Esplike** [comment]
I'll have a look into it because I don't want to break the recipe...

**Comment by Esplike** [comment]
This would break the repipe for ESX because TXAdmin automatically creates the database.

**Pull request closed without merging**